dcor/homogeneity.py

```diff
@@ -185,6 +188,8 @@ def energy_test(*args, **kwargs):
         Exponent of the Euclidean distance, in the range :math:`(0, 2)`.
     random_state: {None, int, array_like, numpy.random.RandomState}
         Random state to generate the permutations.
+    average: Callable[[ArrayLike], float]
```

Comment:
Add the average parameter to the first line of the docstring too, until we merge this function and the `_impl` one and could get rid of that.